common/models/transducer/multi_stager.py

```diff
@@ -0,0 +1,284 @@
+"""
+Usage:
+After each stage alignments are automatically generated.
+
+# dummy
+st1 = Stage(
+  make_net=Pretrain(make_net, {"enc_lstm_dim": (512, 1024), "enc_num_layers": (3, 6)}, num_epochs=5).get_network,
+  stage_epochs=5,
+  fixed_path=False,
+  alignment_topology=rnnt_topology)
+st2 = Stage(
+  make_net=Pretrain(make_net, {"enc_lstm_dim": (512, 1024), "enc_num_layers": (3, 6)}, num_epochs=15).get_network,
+  stage_epochs=15,
+  fixed_path=True,
+  alignment_topology=rnnt_topology)
+
+# Multi stage training with pretraining
+get_network = MultiStager([st1, st2]).get_network
+
+  TODO:
+    - [ ] How to save the information about the differences between alignments of different topologies.
+    - [ ] Make sure that the alignments correspond to the dataset used(sequence_ordering, ..)
+    - [ ] Reset option
+    - [ ] How to define loops? Instead of creating Stages manually (could do a for loop)
+"""
+
+from __future__ import annotations
+from ast import Str
```

Comment:
Why do you need that?

Reply:
don't know how it ended up there.

Reply:
But then please remove it.

Also check PyCharm warnings. It should tell you when some import is unused.